docs/gecko.rst

```diff
@@ -38,19 +38,19 @@ Create An FTL Translation Resource
 Translations for L20n are stored in ``.ftl`` files.  You can find more 
 information about the FTL format at the end of this document.
```

Comment:
This still refers to the inline ftl section we removed.

Move the link up to here?